packages/hs-react-ui/src/components/Table/Table.tsx

```diff
@@ -155,12 +155,12 @@ export const Row = styled(TR)`
       column-gap: ${columnGap};
       position: relative;
       background-color: ${background};
-      height: ${isCollapsed ? '0px' : '100%'};
-  
+      height: ${isCollapsed ? '0px' : undefined};
+
```

Comment:
Is there a way to write this like: 
```suggestion
${isCollapsed && 'height: 0px;'}
```
so we don't assign an attribute to undefined?